### DISCUSSION TITLE
Exporting URLs to SVG and PDF

### DISCUSSION THREAD

**Discussion by k360git** [freeplane/freeplane#1757] [Help] [answered]
Thank you for this amazing tool! I have only recently discovered it.

**Is it possible to export a link that is present in my mind map when I generate an SVG or PDF?** I think I've tried everything but I can't figure it out. Upon inspecting the source of an SVG export I could only see something like `URL-clip` but no `<a href` etc where the link was supposed to be.

It's possible to add it manually of course, at least for SVG exports, but I'm planning to do this often and each map may have a lot of links so it won't be practical.

**Comment by euu2021** [comment, marked as answer]
duplicate of #149
